apps/ftu/js/navigation.js

```diff
@@ -241,6 +241,18 @@ var Navigation = {
         var improve = document.getElementById('browser_os_improve');
         navigator.mozL10n.setAttributes(improve, 'helpImprove2',
           getLocalizedLink('helpImprove'));
+
+        // Initialize the share checkbox according to the preset value
+        // of debug.performance_data.shared
+        var sharePerformance = document.getElementById('share-performance');
+        var setting = sharePerformance.name;
+        var settings = navigator.mozSettings;
```

Comment:
we can make the `settings` var in line 71 [https://github.com/marshall/gaia/blob/bug1109422_mauChanges/apps/ftu/js/navigation.js#L71] global (out of the object), so we can avoid create and initialize again.